integration_tests/02_controller_daemon.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+my_dir="$(dirname "$0")"
+source "$my_dir/header.sh"
+
+export DEALBOT_MINER_ADDRESS=t01000
+
+function finish {
+	kill -9 $CONTROLLER_PID $DAEMON_PID $CONTROLLER_TAIL_PID $DAEMON_TAIL_PID
+}
+trap finish EXIT
+
+dealbot controller --listen="localhost:8085" &>controller.log &
```

Comment:
move below variable definition and use `$CONTROLLER_ENDPOINT`

Reply:
done!